src/renderer/ssf-api.ts

```diff
@@ -674,6 +681,30 @@ export class SSFApi {
       notificationId,
     });
   }
+
+  /**
+   * Get zoom level
+   *
+   * @param {string} locale - language identifier and a region identifier
```

Comment:
Please remove @param and @example as they don't correspond to this method

Reply:
@sbenmoussati 
I updated it.
Thanks!